README.rst

```diff
@@ -109,6 +109,12 @@ Optional arguments:
   be raised if the file can't be read.
   Set to ``False`` to disable this behaviour.
 
+* ``sock`` -- an open socket or socket-like object (such as a `paramiko.Channel`) to use
+  for communication to the target host.  One could setup a proxy_command using this :
+
+    sock=paramiko.proxy.ProxyCommand("ssh -q -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
+                                     "{proxy_user}@{proxy_ip} nc -q0 {target_ip} 22")
```

Comment:
Looking at the rendered output in GitHub, this doesn't quite to seem to render properly. The simplest thing to do would be to use `.. code-block:: python` to get it to formatted as Python code.